Detections/ThreatIntelligenceIndicator/IPEntity_AppServiceHTTPLogs.yaml

```diff
@@ -0,0 +1,63 @@
+id: f9949656-473f-4503-bf43-a9d9890f7d08
+name: (Preview) TI map IP entity to AppServiceHTTPLogs
+description: |
+  'Identifies a match in AppServiceHTTPLogs from any IP IOC from TI'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
+    dataTypes:
+      - ThreatIntelligenceIndicator
+  - connectorId: ThreatIntelligenceTaxii
+    dataTypes:
+      - ThreatIntelligenceIndicator
+  - connectorId: AzureMonitor(AppServices)
+    dataTypes:
+      - AppServiceHTTPLogs
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+query: |
+
+  let dt_lookBack = 1h;
+  let ioc_lookBack = 14d;
+  ThreatIntelligenceIndicator
+  | where TimeGenerated >= ago(ioc_lookBack) and ExpirationDateTime > now()
+  | where Active == true
+  // Picking up only IOC's that contain the entities we want
+  | where isnotempty(NetworkIP) or isnotempty(EmailSourceIpAddress) or isnotempty(NetworkDestinationIP) or isnotempty(NetworkSourceIP)
+  // As there is potentially more than 1 indicator type for matching IP, taking NetworkIP first, then others if that is empty.
+  // Taking the first non-empty value based on potential IOC match availability
+  | extend TI_ipEntity = iff(isnotempty(NetworkIP), NetworkIP, NetworkDestinationIP)
+  | extend TI_ipEntity = iff(isempty(TI_ipEntity) and isnotempty(NetworkSourceIP), NetworkSourceIP, TI_ipEntity)
+  | extend TI_ipEntity = iff(isempty(TI_ipEntity) and isnotempty(EmailSourceIpAddress), EmailSourceIpAddress, TI_ipEntity)
+  | join (
+      AppServiceHTTPLogs | where TimeGenerated >= ago(dt_lookBack)
+    | where isnotempty(CIp)
+    | extend WebApp = split(_ResourceId, '/')[8]
+    // renaming time column so it is clear the log this came from
+    | extend AppService_TimeGenerated = TimeGenerated
+  )
+  on $left.TI_ipEntity == $right.SingleIP
```

Comment:
There is no SingleIP from the right side (AppServiceHTTPLogs) so the join fails, this looks like it should be CIp.